nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -106,26 +69,17 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_HOSTNAME = new PropertyDescriptor.Builder()
-            .name("SMTP_HOSTNAME")
-            .displayName("SMTP hostname")
-            .description("The hostname to be embedded into the banner displayed when an " +
-                    "SMTP client connects to the processor TCP port .")
-            .required(true)
-            .expressionLanguageSupported(false)
-            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .build();
-
-    protected static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
             .name("SMTP_MAXIMUM_CONNECTIONS")
             .displayName("Maximum number of SMTP connection")
             .description("The maximum number of simultaneous SMTP connections.")
             .required(true)
+            .defaultValue("1")
             .expressionLanguageSupported(false)
             .addValidator(StandardValidators.INTEGER_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_TIMEOUT = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_TIMEOUT = new PropertyDescriptor.Builder()
```

Comment:
If you are intending to speak SMTP tgan you need to follow the relevant RFC, this means a connecting MTA should be actively informed of a time out otherwise it will indefinitely hang connected as you do not return a 4XX code nor an ok,  instead we just leave the  connection for dead. This means all clients will have to wait for their own timeouts before disconnecting